instrumentation-tests/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'com.mapbox.maps.token'
+apply from: "${rootDir}/gradle/ktlint.gradle"
+
+android {
+    compileSdkVersion androidVersions.compileSdkVersion
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    defaultConfig {
+        applicationId "com.mapbox.navigation.instrumentation_tests"
+        minSdkVersion androidVersions.minSdkVersion
+        targetSdkVersion androidVersions.targetSdkVersion
+        multiDexEnabled true
+        versionCode 1
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
do you want to try and see if jacoco can update test coverage here?

`AndroidJacocoTestRunner`
http://www.qaautomated.com/2016/03/how-to-find-code-coverage-with-jacoco.html

I'm not really sure how we'd do it either, but posted a support ticket with codecov to see if they have a recommendation https://codecov.freshdesk.com/support/tickets/3970

Reply:
@kmadsen it's definitely possible, we did that in the past - https://github.com/mapbox/mapbox-gl-native/pull/14157. Let's explore it in a follow-up.